sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -740,13 +740,13 @@ CREATE UNIQUE INDEX `uuid` ON `users` (`uuid`);
 ALTER TABLE `uuid_registry` ADD `table_vertical` varchar(255) NOT NULL DEFAULT '';
 #EndIf
 
```

Comment:
This block needs to be added back:
```
#IfMissingColumn facility_user_ids uuid
ALTER TABLE `facility_user_ids` ADD `uuid` binary(16) DEFAULT NULL;
#EndIf

#IfUuidNeedUpdateVertical facility_user_ids uid:facility_id
#EndIf

#IfNotIndex facility_user_ids uuid
CREATE INDEX `uuid` ON `facility_user_ids` (`uuid`);
#EndIf
```

Reply:
Will never do Merge's on Github Confuses me a lot :confused: